pubspec.lock

```diff
@@ -415,4 +415,4 @@ packages:
     source: hosted
     version: "2.1.13"
 sdks:
-  dart: ">=2.0.0-dev <=2.0.0-dev.24.0"
+  dart: ">=2.0.0-dev <=2.0.0-edge.0d5cf900b021bf5c9fa593ffa12b15bcd1cc5fe0"
```

Comment:
You'll likely want something like `2.0.0-dev.30.0` instead of a local build number.

Reply:
done